feat(iOS): Remove GCDWebServer

jcesarmobile opened the pull request:
Removes the GCDWebServer and simplifies code. It requires iOS 11+

BREAKING CHANGE: Sets deployment-target to 11, so will only work on iOS 11+

mlynch commented on README.md:
````diff
@@ -40,19 +40,20 @@ Note: This repo and its documentation are for `cordova-plugin-ionic-webview` @ `
 ```
 
-The default port the server will listen on. _You should change this to a random port number!_
+The default port the server will listen on. If you change it, add an `allow-navigation`
````
Why are there docs for ports now that the server is gone?

mlynch commented on README.md:
```diff
@@ -144,7 +118,11 @@ Whether to use a dark styled keyboard on iOS
-    * The origin for requests from the Web View is `http://localhost:8080`.
+    * The default origin for requests from the WebView is `http://localhost:8080`. If `WKPort` preference is set, then origin will be  `http://localhost:WKPortValue`.
+
+1. Apps are now served from `ionic://` scheme on iOS.
```
This section is a little confusing. Can you more clearly mark that the port stuff is Android only? On that note: should `WKPort` be removed and instead just use a setting like `AndroidHostname` which they could add the port if they want?

mlynch commented on README.md:
```diff
@@ -75,16 +76,6 @@ Other possible values are `1` (`MIXED_CONTENT_NEVER_ALLOW`) and `2` (`MIXED_CONT
-
-On iOS 10 and older will continue using the local web server even if the preference is set to `true`.
-
 #### HostName
```
Hostname is one word, so the setting should be `Hostname`. However, if it's Android only perhaps it should be `AndroidHostname`

jcesarmobile commented:
Changed the file scheme to `app-file` as on Android
Kept `ionic://` for the main serving scheme.
Made `localhost` the default Hostname instead of `app` for consistency
Updated docs

kamilbrk commented:
iOS 11 only?

https://github.com/apache/cordova-ios/issues/415#issuecomment-440695674
https://forum.ionicframework.com/t/ionic-webview-plugin-version-compatibility/151218
